app/javascript/controllers/apex_agents_controller.js

```diff
@@ -0,0 +1,53 @@
+import { Controller } from "@hotwired/stimulus"
```

Comment:
agents will be fleshed out in a different PR. just setting up the controller here.